api/swagger.yaml

```diff
@@ -269,6 +269,17 @@ paths:
       tags:
       - "kubernetes"
       summary: "Create a new Kubernetes account (provider)"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Object that describes the kubernetes provider"
+        required: true
+        schema:
+          $ref: "#/definitions/Provider"
```

Comment:
Rename this to `KubernetesProvider` to allow for other providers in the future.

Reply:
done